processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessorSubscription.java

```diff
@@ -246,24 +261,26 @@ public void run() {
             } catch (RuntimeException e) {
                 log.error("Offset commit failed before closing consumer", e);
             }
-            contexts.close();
-
             processors.destroySingletonScope(scope.subscriptionId());
```

Comment:
If we keep this line, `processors.destroySingletonScope(scope.subscriptionId());` will be called twice upon close. Is that safe?

Reply:
Good catch... I was intend to leave this and do not include it in `cleanUp`